marconi-chain-index/src/Marconi/ChainIndex/Indexers/Utxo.hs

```diff
@@ -252,23 +255,22 @@ data UtxoRow = UtxoRow
   , _urSpentInfo     :: !(Maybe SpentInfo)
   } deriving (Show, Eq, Ord, Generic)
 
-
 $(makeLenses ''UtxoRow)
 
+urSpendSlotNo :: Traversal' UtxoRow C.SlotNo
```

Comment:
should probably be changed from `urSpendSlotNo` to `urSpentSlotNo` to remain consistent.  And replace other occurances of "\*spend\*" with "\*spent\*"